src/index.js

```diff
@@ -55,6 +56,7 @@ export class RootComponent extends React.Component {
 		super( props );
 		setupLocale( props.locale, props.translations );
 		setupApiFetch();
+		setupJetpackEditor( props.jetpackState || { blogId: 1, isJetpackActive: true } );
```

Comment:
We'll have to make this async when we actually fetch the data from the site, but for now let's keep it that way 👍

Reply:
Sounds good. Just going to note here that if we make an async call to check if we're enabling jetpack blocks, then we need to block the action of parsing raw html to blocks until after that function returns, otherwise jetpack blocks will show up as not enabled, even if api response says they should be enabled.